dkms.conf

```diff
@@ -1,19 +1,18 @@
-# auto-load via /etc/modules-load.d/xpmem.conf
-# install udev rule to control /dev file permissions
-
-
 # DKMS module name and version
 PACKAGE_NAME="xpmem"
 PACKAGE_VERSION="2.7.3"
 
+# Need to have a configure script to build first.
+PRE_BUILD="autogen.sh"
+
 # module name
 BUILT_MODULE_NAME[0]="xpmem"
 # where we find the .ko file under the build directory
 BUILT_MODULE_LOCATION[0]="kernel"
 # where we put it under the kernel modules directory
 DEST_MODULE_LOCATION[0]="/kernel/../updates/"
 # how to build it
-MAKE[0]="./configure --prefix=/opt/xpmem; make clean; make install"
+MAKE[0]="./configure --prefix=/opt/xpmem --with-kerneldir=$kernel_source_dir --with-kernelvers=$kernelver ; make"
```

Comment:
I left the `--prefix` thing in there to make this the smallest change, and also least surprise if some bug in autotools causes the library to install when running just `make` (would put it where people usually put it, so minimized harm). I suppose some really random name in `/tmp` would also be suitable, or some place that would be impossible to write to like `/dev/null`.

By the way, I am working on some other changes (future PR) which would allow disabling the build of the library and installing headers entirely so that DKMS here would build the module and not waste any time building the unneeded library.

Reply:
ok for me, but i think it's ok to remove prefix if we don't install

Reply:
It is removed now.